_layouts/help.html

```diff
@@ -9,6 +9,17 @@
   <div class="page-content grid-row">
     <div class="page-content__prose grid-col grid-col-7 help-article">
       {{ page.content | replace: 'site.baseurl', site.baseurl | markdownify }}
+      {% if page.show_common_issues %}
+      <h2>{% t help_page.subtitle %}</h2>
+      <ul class="help-question-list">
+        {% for question in related_questions offset:1 %}
+        {% if page.url contains parent_url %}
+        <li><a href="{{ site.baseurl}}{{ question.url}}">{{ question.title }}</a>
+        </li>
+        {% endif %}
+        {% endfor %}
+      </ul>
+      {% endif %}
```

Comment:
can we indent inside the `if` and `for`?

Reply:
Sure. I have to turn off Liquid linting to do so. Are you using vscode? Does yours automatically indent nested delimiters? If you have that setting, please share 😄

Reply:
😬 I use Sublime and don't have any linters for this.... but if there are tools that standardize this way, maybe I can just adjust my habits